edx_sga/sga.py

```diff
@@ -215,6 +215,7 @@ def student_view(self, context=None):
         )
         fragment.add_css(_resource("static/css/edx_sga.css"))
         fragment.add_javascript(_resource("static/js/src/edx_sga.js"))
+        fragment.add_javascript(_resource("static/js/src/jquery.tablesorter.min.js"))
```

Comment:
I'm not sure if the javascript build pipeline includes XBlock dependencies. @bdero do you know?